dev-requirements.txt

```diff
@@ -48,46 +48,46 @@ eth-rlp==1.0.1 ; python_version >= "3.9" and python_version < "4"
 eth-tester[py-evm]==0.11.0b2 ; python_version >= "3.9" and python_version < "4"
 eth-typing==3.5.2 ; python_version >= "3.9" and python_version < "4"
 eth-utils==2.3.2 ; python_version >= "3.9" and python_version < "4"
-ethpm-types==0.6.18 ; python_version >= "3.9" and python_version < "4"
-eval-type-backport==0.2.0 ; python_version >= "3.9" and python_version < "3.10"
-evm-trace==0.2.3 ; python_version >= "3.9" and python_version < "4"
+ethpm-types==0.6.19 ; python_version >= "3.9" and python_version < "4"
+eval-type-backport==0.2.2 ; python_version >= "3.9" and python_version < "3.10"
+evm-trace==0.2.4 ; python_version >= "3.9" and python_version < "4"
 evmchains==0.0.13 ; python_version >= "3.9" and python_version < "4"
 exceptiongroup==1.2.2 ; python_version >= "3.9" and python_version < "3.11"
 executing==2.1.0 ; python_version >= "3.9" and python_version < "4"
 filelock==3.16.1 ; python_version >= "3.9" and python_version < "4"
-flask==3.0.3 ; python_version >= "3.9" and python_version < "4"
```

Comment:
should all of these packages be changing version?

Reply:
🤷 that's the output of the requirements locking script. To be honest, I had to tweak the result since it removed a dependency from atxm, so at this point, I'm not sure we can fully trust the script. Regardless, I think that's something we can revisit outside this PR.